ocamlformat.opam

```diff
@@ -29,6 +30,7 @@ depends: [
   "dune" {>= "1.11.1"}
   "fpath"
   "ocaml-migrate-parsetree" {>= "1.3.1"}
+  "ocp-indent" {with-test}
```

Comment:
looks like `ocp-indent` is not installed even with `--with-test`

Reply:
This works for me locally with 4.09 and for the CI in 4.07 and 4.08.

Reply:
I think I managed to fix this. The issue was that a previous opam pin was cached and re-adding a pin did update the source code but not the opam metadata.

Reply:
Great, thanks!